hack/install_provider.sh

```diff
@@ -62,17 +62,36 @@ kubectl -n "${INSTALL_NAMESPACE}" apply \
   -f ./config/rbac/clusterrole.yaml \
   -f ./config/rbac/role_binding.yaml \
   -f ./config/rbac/clusterrole_binding.yaml
-yq w ./config/rbac/ibm-crossplane_clusterrole.yaml "metadata.name" "${PROVIDER_NAME}-ibm-crossplane" |\
-  kubectl -n "${INSTALL_NAMESPACE}" apply -f -
-yq w ./config/rbac/ibm-crossplane_clusterrole_binding.yaml "metadata.name" "${PROVIDER_NAME}-ibm-crossplane" |\
-  yq w - "roleRef.name" "${PROVIDER_NAME}-ibm-crossplane" |\
-  kubectl -n "${INSTALL_NAMESPACE}" apply -f -
+if yq --version | grep -q 'version 4'; then
+  echo "[INFO] detected yq version 4"
+  yq e ".metadata.name = \"${PROVIDER_NAME}"\" config/rbac/ibm-crossplane_clusterrole.yaml |\
```

Comment:
@arturobrzut 
I changed naming, because `PROVIDER_NAME` is already set to `ibm-crossplane-provider-ibm-cloud` so naming it like "${PROVIDER_NAME}-ibm-crossplane" doesn't make much sense in my opinion.

Reply:
"${PROVIDER_NAME}-ibm-crossplane" is necessary because otherwise it will overwrite `ClusterRole` for provider